addons/sale_purchase/models/product_template.py

```diff
@@ -9,23 +9,26 @@ class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
     service_to_purchase = fields.Boolean(
-        "Subcontract Service",
-        compute='_compute_service_to_purchase', store=True, readonly=False,
+        "Subcontract Service", company_dependent=True,
```

Comment:
Why don't we store that field any more?
<https://github.com/odoo/odoo/blob/d982c4f319ee3d6af7e4820ab37e5d93a80a6544/odoo/fields.py#L440>